src/views/Onboarding/Complete.vue

```diff
@@ -138,6 +138,13 @@ export default {
 </script>
 
 <style lang="scss" scoped>
+
+.welcome {
+  max-width: 640px;
+  margin-left: auto;
+  margin-right: auto;
+}
```

Comment:
I'm kinda curious if maybe we could just use tailwind's classes instead of writing custom styles here. Wdyt?

Reply:
I agree with you. But when I'm changing existing stuff I prefer to do not to change styles if I'm not introducing any changes to them. But nice catch that I duplicated that one, I was almost sure that I extracted in into `components.scss`. I will remove duplication then.

Reply:
sgtm